### PR TITLE
Fix bug #152 simple or to check if __mm_to_in == 0

### PR DESCRIPTION
**Pull request by IanWorley**
This fixes the bug in the the rain and snow amounts which would return 0.00 because of the conversion to metric

### CODE REVIEW

File: weather_command/_builder.py [hunk 1]
```diff
@@ -348,7 +348,7 @@ def _format_date_time(
 
 
 def _format_precip(precip_amount: float | None, units: str) -> str:
-    if not precip_amount:
+    if not precip_amount or _mm_to_in(precip_amount) < 0.01:
         return "0"
```
Comment (sanders41) [inline weather_command/_builder.py:352]: `_mm_to_in` is rounding to 2 decimal places so you don't need to worry about this being less than 0.01. Also the return should be "0.00"

```py
def _format_precip(precip_amount: float | None, units: str) -> str:
    if not precip_amount:
        return "0.00"
```

**Review by sanders41 — CHANGES_REQUESTED** [review]
I am still seeing some of the "0" values. 

Screenshot from 2022-03-21 18-45-02

Along with the requested change here the other thing that needs to be updated to correct this is to change [this line](https://github.com/sanders41/weather-command/blob/e8fda3cd99ed0568b937a6d6b86b81f936bcc906/weather_command/_builder.py#L430) to:

```py
rain = _format_precip(hourly.rain.one_hour, units) if hourly.rain else "0.00"
```

And [this line](https://github.com/sanders41/weather-command/blob/e8fda3cd99ed0568b937a6d6b86b81f936bcc906/weather_command/_builder.py#L431) to:

```py
snow = _format_precip(hourly.snow.one_hour, units) if hourly.snow else "0.00"
```

File: weather_command/_builder.py (hunk 1 above)
Comment (IanWorley) [reply weather_command/_builder.py]: Sorry I must of misread the issue.

**Review by sanders41 — APPROVED** [review]
Looks good. Thanks for this!

**Pull request merged**